src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -23,10 +23,10 @@
         - name: Log in with a service principal using client certificate.
           text: >
             az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p ~/mycertfile.pem --tenant contoso.onmicrosoft.com
-        - name: Log in using a VM's system assigned identity
+        - name: Log in using a VM's system-assigned managed identity
```

Comment:
Use the official name [**system-assigned managed identity**](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview#how-does-the-managed-identities-for-azure-resources-work).